src/hotspot/cpu/x86/macroAssembler_x86_sha.cpp

```diff
@@ -494,7 +494,7 @@ void MacroAssembler::fast_sha256(XMMRegister msg, XMMRegister state0, XMMRegiste
 #ifdef _LP64
 /*
   The algorithm below is based on Intel publication:
-  "Fast SHA-256 Implementations on Intelë Architecture Processors" by Jim Guilford, Kirk Yap and Vinodh Gopal.
+  "Fast SHA-256 Implementations on Intel(R) Architecture Processors" by Jim Guilford, Kirk Yap and Vinodh Gopal.
```

Comment:
Note: There is of course a unicode `®` symbol, which is what it was originally before it was botched here, but I found no reason to keep this, and in the spirit of JDK-8354213, I thought it better to use pure ASCII here.

Reply:
I guess the difference at L.1 in the various files is just the BOM?

Reply:
> I guess the difference at L.1 in the various files is just the BOM?

Yes.